### ISSUE TITLE
Conflict with built-in command?

### ISSUE THREAD

**Issue by widyakumara** [preset/preset#65]
I was trying to use Tailwind with SvelteKit when came across this project [svelte-add/tailwindcss](https://github.com/svelte-add/tailwindcss).

But when I tried running `npx apply svelte-add/postcss`, it does nothing except echoing:

![npx apply svelte-add/postcss](https://user-images.githubusercontent.com/596434/107084655-e9c6cd80-6829-11eb-9600-1282008aae64.png)

After a few minutes trying, I realised that this could possibly mean that `npx apply` was actually running a built in command (instead of running preset/cli). So I tried `man apply`. And lo!

![man apply](https://user-images.githubusercontent.com/596434/107084707-f9dead00-6829-11eb-82ed-88de5c52b1ca.png)

I don't remember installing it. I also have checked with `brew ls` in case it was installed as a package dependency of other clis that I actually use, and the result is negative.

I'm running:
* macOS Catalina 10.15.7
* node v14.15.4
* npm/npx v6.14.10
* yarn 1.22.4

Oh by the way, in [npx introductory blog post](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner):

> Calling `npx <command>` when `<command>` isn’t already in your `$PATH` will automatically install a package with that name from the npm registry for you, and invoke it.

In other words, if `<command>` (in this case `apply`) is already in my `$PATH`, then by default `npx` will run that.

So this is the expected behaviour on npx part.

**Comment by babichjacob** [comment]
I experience this problem and never mentioned it because I thought it was just a problem with my own setup. Do `npm install --save-dev apply` before running `npx apply ...` as a workaround. (Then uninstall it later.)

**Comment by innocenzi** [comment]
Hey, yeah, this issue has been reported to me before. This is quite annoying, I'm not exactly sure what's the best way to handle it. 

There are a few workarounds:
- As @babichjacob said, install `apply` locally so `npx` can use it
- Use `use-preset` instead of `apply`, which is an alias on NPM that I maintain

I should probably document that for macOS users.

**Comment by widyakumara** [comment]
Hi,

Just tried `use-preset` but it doesn't work. 

![Screen Shot 2021-02-07 at 21 37 25](https://user-images.githubusercontent.com/596434/107150518-bc486400-6990-11eb-903d-3fc97556edcf.png)

I end up using @babichjacob's solution. Kudos.

@innocenzi :

Too bad since I think this is a very useful project.

According to [this page](https://www.freebsd.org/cgi/man.cgi?query=apply&manpath=4.4BSD+Lite2):

> The apply command appeared in 4.2BSD.

The shown date is `April 4, 1994`. So two things for sure:

- This is not a macOS specific issue.
- The command has been around for a very long time.

**Comment by innocenzi** [comment]
Ah, well, that's unexpected. But after thinking about it, I'd rather advice installing `apply` locally than confusing people about the `use-preset` alias, anyway. Not exactly sure why it doesn't work though.

> This is not a macOS specific issue.

From my understanding, macOS has parts of FBSD and that's where the issue comes from. So any OS based on FBSD will have this problem. Thanks for pinning down the issue, I'll update the docs. 👍

**Comment by innocenzi** [comment]
Note: after reading the [docs](https://www.npmjs.com/package/npx), I found you can use `--ignore-existing` to deny `npx` from looking into `$PATH`.
